diff --git a/node_modules/winston-loki/src/batcher.js b/node_modules/winston-loki/src/batcher.js
index af50c40..196e842 100644
--- a/node_modules/winston-loki/src/batcher.js
+++ b/node_modules/winston-loki/src/batcher.js
@@ -12,7 +12,14 @@ let snappy = false
 class Batcher {
   loadSnappy () {
     return require('snappy')
+  }loadSnappy () {
+  // return require('snappy') <--- DINONAKTIFKAN!
+  // Kita kembalikan fungsi palsu agar tidak crash
+  return {
+    compress: (data) => data,
+    uncompress: (data) => data,
   }
+}
 
   loadUrl () {
     let URL
